### ISSUE TITLE
Newt Gingrich not showing up as presidential candidate in 2012

### ISSUE THREAD

**Issue by arm5077** [18F/openFEC#1051]
A bit of a specific problem on which to file an issue, so my apologies, but in case it's indicative of a larger bug...

Newt Gingrich doesn't show up as a presidential candidate under this query: https://api.open.fec.gov/v1/candidates?sort=name&per_page=500&cycle=2012&office=P

I'm also not seeing him as a candidate, period: https://api.open.fec.gov/v1/candidate/P60003654

But he does show up on the [FEC's site](http://www.fec.gov/fecviewer/CandidateCommitteeDetail.do?candidateCommitteeId=P60003654&tabIndex=3). And it looks like he shows up as a former House candidate: https://api.open.fec.gov/v1/candidates/search?name=gingrich&per_page=20&page=1

**Comment by LindsayYoung** [comment]
Thank you for reporting that! I will look into that right now.

**Comment by LindsayYoung** [comment]
We only have F2Z entries for him on dimcandproperties, so he is showing up
as if he was a ballot candidate and we are filtering him out.

@PaulClark2 I think we are missing some more F1s in the data warehouse, can
you check on that please?

**Comment by noahmanger** [comment]
Has this been resolved yet?

**Comment by LindsayYoung** [comment]
Not yet:
Still no results here- 
https://api.open.fec.gov/v1/candidate/P60003654/?api_key=DEMO_KEY 

The underlying table looks the same to me in staging:
```
fec=> select cand_nm, form_tp from dimcandproperties where cand_sk=10005326;
    cand_nm     | form_tp 
----------------+---------
 GINGRICH, NEWT | F2Z
 GINGRICH, NEWT | F2Z
 GINGRICH, NEWT | F2Z
 GINGRICH, NEWT | F2Z
 GINGRICH, NEWT | F2Z
 GINGRICH, NEWT | F2Z
(6 rows)

```

@PaulClak2, when you get a chance can you check on this with the data warehouse? Thanks

**Comment by PaulClark2** [comment]
All rows for P60003654 are loaded using historical load script on 6/24/2015.

In historical load for the dimcandproperties, the script only load the latest one per 
cand_id,
cand_nm,
cand_st1,
cand_st2,
cand_city,
cand_st,
cand_zip,
cand_status,
cand_ici,
election_yr


The rows with FORM_SK of
1543860 and 1543861 has exactly the same 
cand_id,
cand_nm,
cand_st1,
cand_st2,
cand_city,
cand_st,
cand_zip,
cand_status,
cand_ici,
election_yr

So only FORM_SK=1543861 are loaded, but not FORM_SK=1543860.

**Comment by noahmanger** [comment]
I believe this is something that's going to be fixed with the new data, so I'm moving to the icebox.

**Comment by jmcarp** [comment]
Resolved in #1386.
